docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -0,0 +1,884 @@
+    "\n",
+    "# Construction\n",
+    "asym_model = PowerGridModel(asym_input_data)\n",
+    "\n",
+    "# One-time Asymmetric Power Flow Calculation\n",
+    "asym_result = asym_model.calculate_power_flow(symmetric=False,\n",
+    "    error_tolerance=1e-8,\n",
+    "    max_iterations=20,\n",
+    "    calculation_method=CalculationMethod.newton_raphson)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "d08aaf45",
+   "metadata": {},
+   "source": [
+    " We can print a result dataset of node by converting the array to dataframe and refering a specific attribute. In asymmetric calculations, the results of each phase is presented."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "id": "a581a36e",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "------node voltage result------\n",
+      "             0            1            2\n",
+      "0  6054.809261  6033.110279  6054.482030\n",
+      "1  5669.760420  5342.536191  5804.933216\n",
+      "2  5638.571767  5028.415453  5897.375970\n",
+      "------node angle result------\n",
+      "          0         1         2\n",
+      "0 -0.005248 -2.103279  2.092430\n",
+      "1 -0.043049 -2.143908  2.078903\n",
+      "2 -0.054168 -2.157686  2.092568\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"------node voltage result------\")\n",
+    "print(pd.DataFrame(asym_result[\"node\"][\"u\"]))\n",
+    "print(\"------node angle result------\")\n",
+    "print(pd.DataFrame(asym_result[\"node\"][\"u_angle\"]))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "4f59d0f5",
+   "metadata": {},
+   "source": [
+    "### Batch Asymmetric Power Flow Calculation\n",
+    "\n",
+    "We can use the same method `calculate_power_flow` to calculate a number of asymmetric scenarios in one go. To do this, you need to supply an `update_data` argument. This argument contains a dictionary of 3D update arrays (one array per component type per phase).\n",
+    "\n",
+    "The model uses the current data as the base scenario. For each individual calculation, the model applies each mutation to the base scenario and calculates the power flow.\n",
+    "\n",
+    "**NOTE: after the batch calculation, the original model will be kept unchanged. Internally the program copies the original model to multiple batch models for the calculation.**"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "56850333",
+   "metadata": {},
+   "source": [
+    "#### Independent Batch Dataset\n",
+    "\n",
+    "There are two ways to specify the mutations.\n",
+    "\n",
+    "* For each scenario only specify the objects that are changed in this scenario.\n",
+    "* For each scenario specify all objects that are changed in one or more scenarios.\n",
```

Comment:
There are two ways to specify the mutations. For each scenario:
- only specify the objects that are changed in this specific scenario; or
- specify all objects that are changed in one or multiple scenarios.